src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -486,11 +486,8 @@ public boolean parseOptions(final PrintStream messageStream, final String[] args
             }
 
             final String[] pair = arg.split("=", 2);
-            if (pair.length == 2 && pair[1].isEmpty()) {
-
-                if (i < args.length - 1) {
-                    pair[1] = args[++i];
-                }
+            if (pair.length == 2 && pair[1].isEmpty() && i < args.length - 1) {
```

Comment:
here it seems that it would make more sense to merge the two
```
if (pair.length == 2) 
```
into a single if statements:
```
if (pair.length == 2) {
   if (pair[1].isEmpty() && i < args.length - 1) {
       pair[1] = args[++i];
   }
   if (!parseOption(pair[0], pair[1], false)) {
      messageStream.println();
      usage(messageStream, true);
      return false;
  }
}
```

if you are already changing the code.